src/util.ts

```diff
@@ -168,10 +168,18 @@ export async function checkAvailability(
     await makeResponseEmbed(cache, interaction);
   }
 
-  const results = sourceResults.length > 0 || embedResults.length > 0;
   const status = cache.getStatus();
 
-  if (results) {
+  if (status) {
+    status.forEach((s) => {
+      if (s.status === Status.LOADING) s.status = Status.SUCCESS;
+    });
+    cache.setStatus(status);
+  }
+
+  const numberOfSuccesses = status?.filter((s) => s.status === Status.SUCCESS).length ?? 0;
+
+  if (!!numberOfSuccesses) {
```

Comment:
Personally, I dislike the use of truthy conversion as the intention isn't entirely clear
```suggestion
  if (numberOfSuccesses > 0) {
```